app/cdap/services/UiUtils/UrlGenerator.ts

```diff
@@ -180,3 +180,75 @@ export const getOldCDAPUrl = (navigationObj: any = {}) => {
   url = buildCustomUrl(url, stateParams);
   return url;
 };
+
+const buildCustomUrl = (url, params = {}) => {
```

Comment:
i'm guessing this is copied over from somewhere else

Reply:
yes